docs/ECLStandardLibraryReference/SLR-Mods/BLAS.xml

```diff
@@ -0,0 +1,1224 @@
+  support.</para>
+
+  <sect1 id="BLASTypes">
+    <title>Types</title>
+
+    <para><emphasis role="bold">STD.BLAS.Types<indexterm>
+        <primary>STD.BLAS.Types</primary>
+      </indexterm><indexterm>
+        <primary>BLAS.Types</primary>
+      </indexterm><indexterm>
+        <primary>Types</primary>
+      </indexterm></emphasis></para>
+
+    <informaltable colsep="1" frame="all" rowsep="1">
+      <tgroup cols="2">
+        <tbody>
+          <row>
+            <entry><emphasis>value_t</emphasis></entry>
+
+            <entry>REAL8</entry>
+          </row>
+
+          <row>
+            <entry><emphasis>dimension_t</emphasis></entry>
+
+            <entry>UNSIGNED4</entry>
+          </row>
+
+          <row>
+            <entry><emphasis>matrix_t</emphasis></entry>
+
+            <entry>SET OF REAL8</entry>
+          </row>
+
+          <row>
+            <entry><emphasis>Triangle</emphasis></entry>
+
+            <entry>ENUM(UNSIGNED1, Upper=1, Lower=2)</entry>
+          </row>
+
+          <row>
+            <entry><emphasis>Diagonal</emphasis></entry>
+
+            <entry>ENUM(UNSIGNED1, UnitTri=1, NotUnitTri=2)</entry>
+          </row>
+
+          <row>
+            <entry><emphasis>Side</emphasis></entry>
+
+            <entry>ENUM(UNSIGNED1, Ax=1, xA=2)</entry>
+          </row>
+        </tbody>
+      </tgroup>
+    </informaltable>
+
+    <para>Types for the Block Basic Linear Algebra Sub-programs support</para>
+
+    <para></para>
+  </sect1>
+
+  <sect1 id="ICellFunc">
+    <title>ICellFunc</title>
+
+    <para><emphasis role="bold">STD.BLAS.ICellFunc<indexterm>
+        <primary>STD.BLAS.ICellFunc</primary>
+      </indexterm><indexterm>
+        <primary>BLAS.ICellFunc</primary>
+      </indexterm><indexterm>
+        <primary>ICellFunc</primary>
+      </indexterm>(</emphasis> <emphasis>v, r</emphasis> <emphasis
+    role="bold">,</emphasis> <emphasis>c</emphasis> <emphasis>);</emphasis>
+    <emphasis></emphasis></para>
+
+    <informaltable colsep="1" frame="all" rowsep="1">
+      <tgroup cols="2">
+        <colspec colwidth="80.50pt" />
+
+        <colspec />
+
+        <tbody>
+          <row>
+            <entry><emphasis>v</emphasis></entry>
+
+            <entry>The value</entry>
+          </row>
+
+          <row>
+            <entry><emphasis>r</emphasis></entry>
+
+            <entry>The row ordinal</entry>
+          </row>
+
+          <row>
+            <entry><emphasis>c</emphasis></entry>
+
+            <entry>The column ordinal</entry>
+          </row>
+
+          <row>
+            <entry>Return:<emphasis></emphasis></entry>
+
+            <entry>The updated value</entry>
+          </row>
+        </tbody>
+      </tgroup>
+    </informaltable>
+
+    <para><emphasis role="bold">ICellFunc </emphasis>is the function prototype
+    for Apply2Cells.</para>
+
+    <para>Example:</para>
+
+    <programlisting format="linespecific">IMPORT STD;
+STD.BLAS.ICellFunc(42,2,3);
```

Comment:
This is not a good example.
Use REAL8 my_func(Std.BLAS.Types.value_t v, Std.BLAS.Types.dimension_t x, Std.BLAS.Types.dimension_t y) := 1/v;  //set element to the reciprocal value
as a realistic example.